# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../vdr-dvd/dvd-0.3.6_b03-vdr-1.3.38.patch
# Copyright (C) 2006 - 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -aur -x '*.o' dvd-0.3.6_b03/control-dvd.c dvd-0.3.6_b03-patched/control-dvd.c
--- dvd-0.3.6_b03/control-dvd.c	2005-04-12 18:53:43.000000000 +0200
+++ dvd-0.3.6_b03-patched/control-dvd.c	2006-01-08 16:22:15.000000000 +0100
@@ -54,13 +54,13 @@
 
     player->setController(this);
 
-    cStatus::MsgReplaying(this, "DVD");
+    cStatus::MsgReplaying(this, "DVD", "unknown title", true);
 }
 
 cDvdPlayerControl::~cDvdPlayerControl()
 {
     Hide();
-    cStatus::MsgReplaying(this, NULL);
+    cStatus::MsgReplaying(this, "DVD", NULL, false);
     Stop();
     assert(dvd_active == true);
     dvd_active = false;
@@ -310,7 +310,7 @@
             lastCurrent = lastTotal = -1;
             last_title_buffer[0]=0;
             displayReplay->SetTitle("unknown title");
-            cStatus::MsgReplaying(this, "unknown title");
+            cStatus::MsgReplaying(this, "DVD", "unknown title", true);
         }
 
         if (player) {
@@ -318,7 +318,7 @@
             if ( strcmp(title_buffer,last_title_buffer) != 0 ) {
                 displayReplay->SetTitle(title_buffer);
                 if (!Initial) displayReplay->Flush();
-                cStatus::MsgReplaying(this, title_buffer);
+                cStatus::MsgReplaying(this, "DVD", title_buffer, true);
                 strcpy(last_title_buffer, title_buffer);
             }
         }
@@ -568,7 +568,7 @@
 	        title_buffer = GetDisplayHeaderLine();
             if ( strcmp(title_buffer,last_title_buffer) != 0 ) {
  	            strcpy(last_title_buffer, title_buffer);
- 	            cStatus::MsgReplaying(this, title_buffer);
+ 	            cStatus::MsgReplaying(this, "DVD", title_buffer, true);
             }
         }
     }
